<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefScreenShot</class>
 <widget class="QWidget" name="PrefScreenShot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>448</width>
     <height>128</height>
    </rect>
   </property>
   <property name="title">
    <string>Screenshots</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="use_screenshots_check">
      <property name="text">
       <string>Enable screenshots</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="screenshots_dir_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Folder:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>screenshot_edit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="FileChooser" name="screenshot_edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="screenshot_template_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Template:</string>
        </property>
        <property name="buddy">
         <cstring>screenshot_template_edit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="screenshot_template_edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="screenshot_format_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Format:</string>
        </property>
        <property name="buddy">
         <cstring>screenshot_format_combo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="screenshot_format_combo">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileChooser</class>
   <extends>QLineEdit</extends>
   <header>../smplayer/filechooser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
